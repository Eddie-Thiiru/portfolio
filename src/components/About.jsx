import htmlLogo from "../images/html.svg";
import cssLogo from "../images/css.svg";
import jsLogo from "../images/js.svg";
import reactLogo from "../images/react.svg";
import viteLogo from "../images/vite.svg";
import expressLogo from "../images/express.svg";
import nodeLogo from "../images/node-js.svg";
import vitestLogo from "../images/vitest.svg";
import jestLogo from "../images/jest.svg";
import gitLogo from "../images/git.svg";
import webpackLogo from "../images/webpack.svg";
import npmLogo from "../images/npm.svg";
import "../styles/About.css";

const Grid = () => {
  return (
    <div className="technologiesGrid">
      <div className="technologiesWrapper">
        <div>
          <img src={htmlLogo} alt="" />
          <p>HTML</p>
        </div>
        <div>
          <img src={cssLogo} alt="" />
          <p>CSS</p>
        </div>
        <div>
          <img src={jsLogo} alt="" />
          <p>javaScript</p>
        </div>
      </div>
      <div className="technologiesWrapper">
        <div>
          <img src={viteLogo} alt="" />
          <p>Vite</p>
        </div>
        <div>
          <img src={reactLogo} alt="" />
          <p>React</p>
        </div>
        <div>
          <img src={expressLogo} alt="" />
          <p>Express.js</p>
        </div>
      </div>
      <div className="technologiesWrapper">
        <div>
          <img src={nodeLogo} alt="" />
          <p>Node.js</p>
        </div>
        <div>
          <img src={vitestLogo} alt="" />
          <p>Vitest</p>
        </div>
        <div>
          <img src={jestLogo} alt="" />
          <p>Jest</p>
        </div>
      </div>
      <div className="technologiesWrapper">
        <div>
          <img src={gitLogo} alt="" />
          <p>Git</p>
        </div>
        <div>
          <img src={webpackLogo} alt="" />
          <p>Webpack</p>
        </div>
        <div>
          <img src={npmLogo} alt="" />
          <p>npm</p>
        </div>
      </div>
    </div>
  );
};

const About = () => {
  return (
    <section className="aboutSection">
      <h2>-About</h2>
      <p>
        Hello, I am Eddie, a passionate full stack web developer who loves
        solving problems and delivering high quality products.
      </p>
      <Grid />
    </section>
  );
};

export default About;
