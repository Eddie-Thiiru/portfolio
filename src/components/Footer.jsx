const Footer = () => {
  <footer>
    <div className="footerOne">
      <div className="emailWrapper">
        <h2>
          Let&rsquo;s
          <br />
          Connect
        </h2>
        <p>eddiewachaga6@gmail.com</p>
      </div>
      <div className="formWrapper">
        <form>
          <label>
            Your name:
            <input type="text" />
          </label>
          <label>
            Your email address:
            <input type="email" />
          </label>
          <label>
            Your message:
            <input type="text" />
          </label>
          <button type="submit" className="msgBtn">
            Send
          </button>
        </form>
      </div>
    </div>
    <div className="footerTwo">
      <nav className="socialLinks">
        <a>
          <img src="" alt="" />
        </a>
        <a>
          <img src="" alt="" />
        </a>
        <a>
          <img src="" alt="" />
        </a>
      </nav>
      <p>copyright</p>
    </div>
  </footer>;
};

export default Footer;
