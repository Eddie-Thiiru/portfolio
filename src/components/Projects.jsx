import PropTypes from "prop-types";
import { BsGithub } from "react-icons/bs";
import { FiArrowRight, FiArrowUpRight } from "react-icons/fi";
import vaultImg from "../images/game-store-screenshot.png";
import cvImg from "../images/cvBuilder-screenshot.png";
import todoImg from "../images/todo-list-screenshot.png";
import "../styles/Projects.css";

const ProjectCard = ({
  title,
  description,
  liveSite,
  github,
  imageURL,
  className,
}) => {
  return (
    <div className={`${className} project`}>
      <header className="projectHeader">
        <div>
          <h3>{title}</h3>
          <p>{description}</p>
        </div>
        <nav className="projectLinks">
          <a href={liveSite} title="Live Site">
            <FiArrowUpRight />
          </a>
          <a href={github} title="Github">
            <BsGithub />
          </a>
        </nav>
      </header>
      <div className="projectImgWrapper">
        <img src={imageURL} alt="" className="projectScreenshot1" />
        <img src={imageURL} alt="" className="projectScreenshot2" />
      </div>
    </div>
  );
};

const Projects = () => {
  return (
    <section className="projectsSection">
      <h2>Projects</h2>
      <ProjectCard
        title="Game Vault"
        description="Video game store"
        liveSite=""
        github="https://github.com/Eddie-Thiiru/shopping-cart"
        imageURL={vaultImg}
        className="project1"
      />
      <ProjectCard
        title="cvBuilder"
        description="Online CV builder app"
        liveSite="https://eddie-thiiru-cvbuilder.netlify.app/"
        github="https://github.com/Eddie-Thiiru/cv-application"
        imageURL={cvImg}
        className="project2"
      />
      <ProjectCard
        title="ToDo Tasks"
        description="To-do list app"
        liveSite=""
        github="https://github.com/Eddie-Thiiru/todo-list"
        imageURL={todoImg}
        className="project3"
      />
      <a href="https://github.com/Eddie-Thiiru?tab=repositories">
        Explore more <FiArrowRight />
      </a>
    </section>
  );
};

ProjectCard.propTypes = {
  title: PropTypes.string,
  description: PropTypes.string,
  liveSite: PropTypes.string,
  github: PropTypes.string,
  imageURL: PropTypes.string,
  className: PropTypes.string,
};

export default Projects;
