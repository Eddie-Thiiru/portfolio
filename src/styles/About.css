.aboutSection {
  min-height: calc(100vh - 70px);
  display: grid;
  justify-items: center;
  align-items: center;
}

.aboutSection > p {
  width: clamp(45ch, 50%, 75ch);
  justify-self: center;
  text-align: center;
}

.technologiesGrid {
  width: 80%;
  display: grid;
  grid-template-columns: repeat(4, minmax(100px, 1fr));
}

.technologiesWrapper {
  display: flex;
  flex-direction: column;
}

.technologiesWrapper div {
  margin: 20px;
  display: flex;
  flex-direction: column;
  align-items: center;
  gap: 5px;
}

.technologiesGrid img {
  width: 70px;
}
