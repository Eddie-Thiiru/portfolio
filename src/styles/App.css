.App {
  display: grid;
  grid-template-columns: 1fr 5fr 1fr;
  /* grid-template-rows: 70px repeat(aut0-fit, minmax(max-content, 1fr)); */
}

.App > header,
.App > section,
.App > footer {
  grid-column: 2 / 3;
  padding: 10px;
}

nav {
  padding: 5px;
  display: flex;
}

h1,
h2,
h3,
p,
a {
  padding: 5px;
}

a {
  font-weight: 500;
  color: #646cff;
  display: flex;
  align-items: center;
  gap: 2px;
}

a.active {
  color: red;
}
