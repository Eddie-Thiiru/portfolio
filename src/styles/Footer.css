.contactSection {
  --contentWidth: 450px;
  display: flex;
  flex-direction: column;
  padding: 50px max(2rem, 50vw - var(--contentWidth) / 2);
}

.footerOne {
  flex: 1;
  display: flex;
  align-items: center;
  flex-wrap: wrap;
  gap: 50px;
}

.footerOne > div {
  flex: 1;
  display: flex;
  flex-direction: column;
  align-items: center;
}

.footerTwo {
  display: flex;
  flex-direction: column;
  align-items: center;
}

form,
label {
  display: flex;
  flex-direction: column;
}

.sendBtn {
  font-size: 1rem;
  border: none;
  border-radius: 2px;
  margin: 20px 10px;
  padding: 6px;
  display: flex;
  align-items: center;
  align-self: flex-start;
  gap: 5px;
}

label {
  padding: 10px;
}

input {
  font-size: 1rem;
  border: none;
  border-bottom: 2px solid #2b3544;
  margin-top: 10px;
  padding-top: 5px;
  color: #f9fafb;
  background-color: #212934;
}

.emailWrapper h2,
.emailWrapper p {
  min-width: 215px;
}

@media (min-width: 1024px) {
  .contactSection {
    --contentWidth: 1024px;
  }

  input {
    width: 300px;
  }
}

@media (min-width: 1440px) {
  .contactSection {
    --contentWidth: 1440px;
  }
}

.contactSection h2 {
  font-size: clamp(2.5rem, 4vw, 6rem);
}

.contactSection p {
  color: #646cff;
}

.contactSection a {
  background-color: #1a2029;
}

input:focus {
  outline: none;
}

label:focus-within {
  color: #596e90;
}

.sendBtn:hover {
  background-color: #b3b3b3;
}
