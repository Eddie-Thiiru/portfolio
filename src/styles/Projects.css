.projectsSection {
  display: flex;
  flex-direction: column;
  align-items: center;
  gap: 40px;
}

.projectsSection > div {
  padding: 10px;
  aspect-ratio: 1;
}

.project1 {
  grid-column: 2 / 3;
  grid-row: 1 / 3;
  background-color: lightblue;
}

.project2 {
  grid-column: 1 / 2;
  grid-row: 2 / 4;
  background-color: lightblue;
}

.project3 {
  grid-column: 2 / 3;
  grid-row: 3 / 5;
  background-color: lightblue;
}

.projectHeader {
  display: grid;
  grid-template-columns: 1fr 1fr;
  align-items: start;
}

.projectLinks {
  justify-self: end;
  gap: 15px;
}

.projectHeader p {
  font-size: 0.875rem;
}

.projectsSection a {
  grid-row: 4 / 5;
  align-self: center;
}

.projectLinks a {
  height: 30px;
  width: 30px;
  border-radius: 50px;
  background-color: white;
  justify-content: center;
}

.projectLinks svg {
  height: 15px;
  width: 15px;
}

@media (min-width: 450px) {
  .projectsSection {
    display: grid;
    grid-template-columns: repeat(2, minmax(250px, 30%));
    grid-auto-rows: 1fr;
    justify-content: center;
  }
}

@media (min-width: 1024px) {
  .projectHeader p {
    font-size: 1rem;
  }
}

@media (min-width: 1440px) {
  .projectLinks a {
    height: 35px;
    width: 35px;
  }

  .projectLinks svg {
    height: 20px;
    width: 20px;
  }
}
