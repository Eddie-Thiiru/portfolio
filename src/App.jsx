import LandingPage from "./components/Landing";
import About from "./components/About";
import Projects from "./components/Projects";
import Footer from "./components/Footer";
import "./App.css";

function App() {
  return (
    <div className="App">
      <LandingPage />
      <About />
      <Projects />
      <Footer />
    </div>
  );
}

export default App;
