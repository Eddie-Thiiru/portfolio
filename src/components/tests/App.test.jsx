import { describe, expect, it, vi } from "vitest";
import { screen, render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import App from "../../App";
import LandingPage from "../Landing";
import About from "../About";
import Projects from "../Projects";
import Footer from "../Footer";

vi.mock("react-router-dom", async () => {
  const actual = await vi.importActual("react-router-dom");

  return {
    ...actual,

    ScrollRestoration: () => vi.fn(),
  };
});

describe("App component", () => {
  it("should render app", () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });

  it("should render landing page", () => {
    render(
      <BrowserRouter>
        <LandingPage />
      </BrowserRouter>,
    );

    expect(screen.getByRole("heading", { level: 1 }).textContent).toMatch(
      /et/i,
    );
    expect(screen.getByRole("heading", { level: 1 }).textContent).toMatch(
      /eddiethiiru/i,
    );
    expect(screen.getByText("About")).toBeInTheDocument();
    expect(screen.getByText("Projects")).toBeInTheDocument();
    expect(screen.getByText("Contact")).toBeInTheDocument();
  });

  it("should render About section", () => {
    render(
      <BrowserRouter>
        <About />
      </BrowserRouter>,
    );

    expect(screen.getByRole("heading", { name: "-About" })).toBeInTheDocument();
  });

  it("should render projects section", () => {
    render(
      <BrowserRouter>
        <Projects />
      </BrowserRouter>,
    );

    expect(
      screen.getByRole("heading", { name: "-Projects" }),
    ).toBeInTheDocument();
    expect(screen.getAllByRole("heading", { level: 3 })).toHaveLength(3);
    expect(screen.getAllByRole("link", { name: "Live Site" })).toHaveLength(3);
    expect(screen.getAllByRole("link", { name: "Github" })).toHaveLength(3);
    expect(
      screen.getByRole("link", { name: "Explore more" }),
    ).toBeInTheDocument();
  });

  it("should render footer", () => {
    render(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>,
    );

    expect(screen.getByRole("heading").textContent).toMatch(/let'sconnect/i);
    expect(screen.getByRole("button", { name: "Send" })).toBeInTheDocument();
    expect(screen.getByText(/eddiewachaga6@gmail.com/i)).toBeInTheDocument();
    expect(screen.getByText(/copyright/i)).toBeInTheDocument();
  });
});
