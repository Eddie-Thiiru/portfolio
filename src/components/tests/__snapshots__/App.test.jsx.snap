// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App component > should render app 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="mainHeader"
    >
      <h2>
        ET
      </h2>
      <nav
        class="headerNav"
      >
        <a>
          About
        </a>
        <a>
          Projects
        </a>
        <a>
          Contact
        </a>
      </nav>
    </header>
    <section
      class="heroSection"
    >
      <h1>
        Eddie
        <br />
        Thiiru
      </h1>
      <h2>
        Full Stack
        <br />
        Developer
      </h2>
    </section>
    <section
      class="aboutSection"
    >
      <h2>
        -About
      </h2>
      <p>
        Hello, I am Eddie, a passionate full stack web developer who loves solving problems and delivering high quality products.
      </p>
      <div
        class="technologiesGrid"
      >
        <div
          class="technologiesWrapper"
        >
          <div>
            <img
              alt=""
              src="/src/images/html.svg"
            />
            <p>
              HTML
            </p>
          </div>
          <div>
            <img
              alt=""
              src="/src/images/css.svg"
            />
            <p>
              CSS
            </p>
          </div>
          <div>
            <img
              alt=""
              src="/src/images/js.svg"
            />
            <p>
              javaScript
            </p>
          </div>
        </div>
        <div
          class="technologiesWrapper"
        >
          <div>
            <img
              alt=""
              src="/src/images/vite.svg"
            />
            <p>
              Vite
            </p>
          </div>
          <div>
            <img
              alt=""
              src="/src/images/react.svg"
            />
            <p>
              React
            </p>
          </div>
          <div>
            <img
              alt=""
              src="/src/images/express.svg"
            />
            <p>
              Express.js
            </p>
          </div>
        </div>
        <div
          class="technologiesWrapper"
        >
          <div>
            <img
              alt=""
              src="/src/images/node-js.svg"
            />
            <p>
              Node.js
            </p>
          </div>
          <div>
            <img
              alt=""
              src="/src/images/vitest.svg"
            />
            <p>
              Vitest
            </p>
          </div>
          <div>
            <img
              alt=""
              src="/src/images/jest.svg"
            />
            <p>
              Jest
            </p>
          </div>
        </div>
        <div
          class="technologiesWrapper"
        >
          <div>
            <img
              alt=""
              src="/src/images/git.svg"
            />
            <p>
              Git
            </p>
          </div>
          <div>
            <img
              alt=""
              src="/src/images/webpack.svg"
            />
            <p>
              Webpack
            </p>
          </div>
          <div>
            <img
              alt=""
              src="/src/images/npm.svg"
            />
            <p>
              npm
            </p>
          </div>
        </div>
      </div>
    </section>
    <section
      class="projectsSection"
    >
      <h2>
        -Projects
      </h2>
      <div
        class="project1"
      >
        <header
          class="projectHeader"
        >
          <div>
            <h3>
              cvBuilder
            </h3>
            <p>
              Online CV builder app
            </p>
          </div>
          <nav
            class="projectLinks"
          >
            <a
              href="https://eddie-thiiru-cvbuilder.netlify.app/"
            >
              Live Site 
              <svg
                fill="none"
                height="1em"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <line
                  x1="7"
                  x2="17"
                  y1="17"
                  y2="7"
                />
                <polyline
                  points="7 7 17 7 17 17"
                />
              </svg>
            </a>
            <a
              href=""
            >
              Github 
              <svg
                fill="none"
                height="1em"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <line
                  x1="7"
                  x2="17"
                  y1="17"
                  y2="7"
                />
                <polyline
                  points="7 7 17 7 17 17"
                />
              </svg>
            </a>
          </nav>
        </header>
        <div
          class="projectImgWrapper"
        >
          <img
            alt=""
            src=""
          />
        </div>
      </div>
      <div
        class="project2"
      >
        <header
          class="projectHeader"
        >
          <div>
            <h3>
              cvBuilder
            </h3>
            <p>
              Online CV builder app
            </p>
          </div>
          <nav
            class="projectLinks"
          >
            <a
              href="https://eddie-thiiru-cvbuilder.netlify.app/"
            >
              Live Site 
              <svg
                fill="none"
                height="1em"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <line
                  x1="7"
                  x2="17"
                  y1="17"
                  y2="7"
                />
                <polyline
                  points="7 7 17 7 17 17"
                />
              </svg>
            </a>
            <a
              href=""
            >
              Github 
              <svg
                fill="none"
                height="1em"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <line
                  x1="7"
                  x2="17"
                  y1="17"
                  y2="7"
                />
                <polyline
                  points="7 7 17 7 17 17"
                />
              </svg>
            </a>
          </nav>
        </header>
        <div
          class="projectImgWrapper"
        >
          <img
            alt=""
            src=""
          />
        </div>
      </div>
      <div
        class="project3"
      >
        <header
          class="projectHeader"
        >
          <div>
            <h3>
              cvBuilder
            </h3>
            <p>
              Online CV builder app
            </p>
          </div>
          <nav
            class="projectLinks"
          >
            <a
              href="https://eddie-thiiru-cvbuilder.netlify.app/"
            >
              Live Site 
              <svg
                fill="none"
                height="1em"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <line
                  x1="7"
                  x2="17"
                  y1="17"
                  y2="7"
                />
                <polyline
                  points="7 7 17 7 17 17"
                />
              </svg>
            </a>
            <a
              href=""
            >
              Github 
              <svg
                fill="none"
                height="1em"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <line
                  x1="7"
                  x2="17"
                  y1="17"
                  y2="7"
                />
                <polyline
                  points="7 7 17 7 17 17"
                />
              </svg>
            </a>
          </nav>
        </header>
        <div
          class="projectImgWrapper"
        >
          <img
            alt=""
            src=""
          />
        </div>
      </div>
      <a
        href=""
      >
        Explore more 
        <svg
          fill="none"
          height="1em"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <line
            x1="5"
            x2="19"
            y1="12"
            y2="12"
          />
          <polyline
            points="12 5 19 12 12 19"
          />
        </svg>
      </a>
    </section>
    <footer
      class="contactSection"
    >
      <div
        class="footerOne"
      >
        <div
          class="emailWrapper"
        >
          <h2>
            Let's
            <br />
            Connect.
          </h2>
          <p>
            eddiewachaga6@gmail.com
          </p>
        </div>
        <div
          class="formWrapper"
        >
          <form>
            <label>
              Your name:
              <input
                type="text"
              />
            </label>
            <label>
              Your email address:
              <input
                type="email"
              />
            </label>
            <label>
              Your message:
              <input
                type="text"
              />
            </label>
            <button
              class="sendBtn"
              type="submit"
            >
              Send
              <svg
                fill="none"
                height="1em"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <line
                  x1="5"
                  x2="19"
                  y1="12"
                  y2="12"
                />
                <polyline
                  points="12 5 19 12 12 19"
                />
              </svg>
            </button>
          </form>
        </div>
      </div>
      <div
        class="footerTwo"
      >
        <nav
          class="socialLinks"
        >
          <a
            href="#"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.012 8.012 0 0 0 16 8c0-4.42-3.58-8-8-8z"
              />
            </svg>
          </a>
          <a
            href="#"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z"
              />
            </svg>
          </a>
          <a
            href="#"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5.026 15c6.038 0 9.341-5.003 9.341-9.334 0-.14 0-.282-.006-.422A6.685 6.685 0 0 0 16 3.542a6.658 6.658 0 0 1-1.889.518 3.301 3.301 0 0 0 1.447-1.817 6.533 6.533 0 0 1-2.087.793A3.286 3.286 0 0 0 7.875 6.03a9.325 9.325 0 0 1-6.767-3.429 3.289 3.289 0 0 0 1.018 4.382A3.323 3.323 0 0 1 .64 6.575v.045a3.288 3.288 0 0 0 2.632 3.218 3.203 3.203 0 0 1-.865.115 3.23 3.23 0 0 1-.614-.057 3.283 3.283 0 0 0 3.067 2.277A6.588 6.588 0 0 1 .78 13.58a6.32 6.32 0 0 1-.78-.045A9.344 9.344 0 0 0 5.026 15z"
              />
            </svg>
          </a>
        </nav>
      </div>
    </footer>
  </div>
</div>
`;
